src/js/views/patients/sidebar/action/action-sidebar-activity-views.js

```diff
@@ -20,93 +20,93 @@ import 'scss/modules/sidebar.scss';
 import './action-sidebar.scss';
 
 const CreatedTemplate = hbs`
-  {{formatHTMLMessage (intlGet "patients.sidebar.action.activityViews.created") name = name team = team}}
+  {{formatHTMLMessage (intlGet (getI18nSource "created")) name = name team = team}}
```

Comment:
This `name = name team = team` syntax doesn't bother anyone? 🙀 Makes me feel like my eyes are not working right... 😅